cpp/include/cuspatial_test/base_fixture.hpp

```diff
@@ -22,15 +23,12 @@ namespace cuspatial {
 namespace test {
 
 /**
- * @brief Base test fixture class from which all libcuspatial tests should inherit.
- *
- * Example:
- * ```
- * class MyTestFixture : public cuspatial::test::BaseFixture {};
- * ```
+ * @brief Uses CRTP to supply rmm resources for test fixtures.
```

Comment:
Why is the CRTP needed here? Where is the template parameter used?

Reply:
Functionally making this class a base class for both test fixtures would also work. Making it as a mixin class is because there may be other classes that can also use this mixin to provide RMM resources.

Reply:
I understand why it should be a mixin. I'm asking why is CRTP needed in order to make it a mixin? It doesn't use anything from the template parameter?

Reply:
Perhaps I misunderstood the use of CRTP in this case. If it doesn't use the template parameter, then there isn't a need for it.